scripts/builddown

```diff
@@ -11,7 +11,7 @@ echo " for advanced usage, see ${scriptdir}/foreach.sh"
 echo "************************************************************"
 
 if [ "$#" -eq 0 ]; then
-  ${scriptdir}/foreach.sh --reset
+  ${scriptdir}/foreach.sh --reset --down yarn build
```

Comment:
Won't this mean that the build is run twice (once here, and once on line 22)? Same question for buildup, obivously.

Reply:
AH i thought i moved the second one into the else statement! thanks for the catch